src/ai_hawk/linkedIn_easy_applier.py

```diff
@@ -535,7 +535,10 @@ def _create_and_upload_resume(self, element, job):
     def _create_and_upload_cover_letter(self, element: WebElement, job) -> None:
         logger.debug("Starting the process of creating and uploading cover letter.")
 
-        cover_letter_text = self.gpt_answerer.answer_question_textual_wide_range("Write a cover letter")
+        cover_letter_text = self.gpt_answerer.answer_question_textual_wide_range(
```

Comment:
you should also pass your resume.

is this question param or prompt? if prompt move this to prompt.py